src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -252,6 +257,11 @@ public async Task<ActionResult> RefreshJwtCookie()
         /// </summary>
         /// <returns>The result of the action. Contains the new token if the old token was valid and could be exchanged.</returns>
         [AllowAnonymous]
+        [Produces("text/plain")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status424FailedDependency)]
```

Comment:
Have you observed any cases where the endpoint returns 424?

Reply:
oops... Seems like I have a typo. https://github.com/Altinn/altinn-studio/blob/be5583af15ecf978a091de3c8fcb063f547a7824/src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs#L507

```suggestion
        [ProducesResponseType(typeof(void), StatusCodes.Status429TooManyRequests)]
```

Reply:
Ah, okay.